vector/src/main/java/im/vector/app/features/spaces/SubSpaceSummaryItem.kt

```diff
@@ -0,0 +1,106 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.spaces
+
+import android.widget.ImageView
+import android.widget.Space
+import android.widget.TextView
+import androidx.core.content.ContextCompat
+import androidx.core.view.isGone
+import androidx.core.view.isVisible
+import androidx.core.view.updateLayoutParams
+import com.airbnb.epoxy.EpoxyAttribute
+import com.airbnb.epoxy.EpoxyModelClass
+import im.vector.app.R
+import im.vector.app.core.epoxy.VectorEpoxyHolder
+import im.vector.app.core.epoxy.VectorEpoxyModel
+import im.vector.app.core.platform.CheckableConstraintLayout
+import im.vector.app.core.utils.DebouncedClickListener
+import im.vector.app.features.home.AvatarRenderer
+import im.vector.app.features.home.room.list.UnreadCounterBadgeView
+import org.matrix.android.sdk.api.util.MatrixItem
+
+@EpoxyModelClass(layout = R.layout.item_sub_space)
+abstract class SubSpaceSummaryItem : VectorEpoxyModel<SubSpaceSummaryItem.Holder>() {
+
+    @EpoxyAttribute(EpoxyAttribute.Option.DoNotHash)  lateinit var avatarRenderer: AvatarRenderer
+    @EpoxyAttribute lateinit var matrixItem: MatrixItem
+    @EpoxyAttribute var selected: Boolean = false
+    @EpoxyAttribute(EpoxyAttribute.Option.DoNotHash) var listener: (() -> Unit)? = null
+    @EpoxyAttribute(EpoxyAttribute.Option.DoNotHash)  var onMore: (() -> Unit)? = null
+    @EpoxyAttribute(EpoxyAttribute.Option.DoNotHash)  var toggleExpand: (() -> Unit)? = null
+    @EpoxyAttribute var expanded: Boolean = false
+    @EpoxyAttribute var hasChildren: Boolean = false
+    @EpoxyAttribute var indent: Int = 0
+    @EpoxyAttribute var countState : UnreadCounterBadgeView.State = UnreadCounterBadgeView.State(0, false)
+
+    override fun bind(holder: Holder) {
+        super.bind(holder)
+        holder.rootView.setOnClickListener { listener?.invoke() }
+        holder.groupNameView.text = matrixItem.displayName
+        holder.rootView.isChecked = selected
+        if (onMore != null) {
+            holder.moreView.isVisible = true
+            holder.moreView.setOnClickListener(
+                    DebouncedClickListener({ _ ->
+                        onMore?.invoke()
+                    })
+            )
+        } else {
+            holder.moreView.isVisible = false
+        }
+
+        if (hasChildren) {
+            holder.collapseIndicator.isVisible = true
+            holder.collapseIndicator.setImageDrawable(
+                    ContextCompat.getDrawable(holder.view.context,
+                            if (expanded) R.drawable.ic_expand_less else R.drawable.ic_expand_more
+                    )
+            )
+            holder.collapseIndicator.setOnClickListener(
+                    DebouncedClickListener({ _ ->
+                        toggleExpand?.invoke()
+                    })
+            )
+        } else {
+            holder.collapseIndicator.isGone = true
```

Comment:
remove from `if else` block and write `holder.collapseIndicator.isVisible = hasChildren` ?

Reply:
updated